### PR TITLE
document how to hack on pg_activity

### LINKED ISSUE #386: Document how to run tests locally
To speed up local development - currently need to push to Github and feedback lags.

Currently unclear I think for the more casual developer. I did try the `tox` command from Github actions YAML...but got a bunch of errors on master which GH shows as clean. Thank you.

### PR DESCRIPTION
**Pull request by dlax**
Fix #386.

### CODE REVIEW

**Comment by kmoppel-cognite** [comment]
Looks good to a more casual developer now :)

File: README.md [hunk 1]
```diff
@@ -242,6 +242,70 @@ Information can also be given via PostgreSQL's environment variables
...
+    $ .venv/bin/activate
+    (.venv) $ pip install -e ".[dev]"
+
+The source code is formatted with [black][] and [isort][] and typed checked with
+[mypy][] (all those are included in the development environment). Make sure to
+respect this, e.g. by configuring your editor, before committing changes.
+Alternatively, you can install [pre-commit][] hooks so that this will be checked
+automatically:
+
+    (.venv) $ pip install -e ".[dev]"
```
Comment (blogh) [inline README.md:271]: is it the correct command ?

Comment (blogh) [inline README.md:271]: should it be `pre-commit install` ?

**Review by blogh — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (dlax) [reply README.md]: Indeed; fixed, thank you.

**Pull request merged**